Makefile

```diff
@@ -114,23 +115,25 @@ _update_crds: update_devworkspace_crds
 	$(TOOL) apply -f ./devworkspace-crds/deploy/crds
 
 _update_controller_configmap:
-	$(TOOL) apply -f ./deploy/controller_config.yaml
+	$(TOOL) apply -f ./deploy/controller_config.yaml -n $(NAMESPACE)
 
 _apply_controller_cfg:
-	$(TOOL) apply -f ./deploy
+	sed -i.bak -e "s|namespace: che-workspace-controller|namespace: $(NAMESPACE)|" ./deploy/role_binding.yaml
```

Comment:
I think it makes sense to change hard-coded namespace to placeholder, like:
```
# The ${NAMESPACE} must be replaced with the actual namespace where controller lives
namespace: ${NAMESPACE}
```
It would help users detect the issue with deployment earlier if namespace is not `che-workspace-controller` since they  will get error message after `oc apply` while without it - deployment is deployed and run but workspace won't be started because of permissions leak (<- I haven't checked that assumption). WDYT?

Reply:
Good idea -- done.